pkg/app/run_test.go

```diff
@@ -263,7 +263,7 @@ COMMANDS
     -n, --name=NAME                Name of package, defaulting to directory name
                                    of the --path destination
     -d, --description=DESCRIPTION  Description of the package
-    -a, --author=AUTHOR            Author of the package
+    -a, --author=AUTHOR ...        Author of the package
```

Comment:
```suggestion
    -a, --author=AUTHOR ...        Author(s) of the package
```
(in case you want to commit both suggestions from the GH UI)